dicom-archive/DICOM/Element.pm

```diff
@@ -204,9 +306,19 @@ sub readSequence {
 }
 
 
-# Return the Value Field length, and length before Value Field.
-# Implicit VR: Length is 4 byte int.
-# Explicit VR: 2 bytes hold VR, then 2 byte length.
+=pod
+
+=head3 readLength($IN)
+
+Reads length.
```

Comment:
Reads the length of a VR from a file, as an integer encoded on 16 or 32 bits.

Reply:
done